### PR TITLE
Add subversion client (svn) plus requirements

### PR DESCRIPTION
**Pull request by msarahan**
ping @ccordoba12 @groutr - quick review if you have time

### CODE REVIEW

File: svn/bld.bat [hunk 1]
```diff
@@ -0,0 +1,31 @@
+if "%ARCH%"=="32" (
+   set "PLATFORM=Win32"
+) else (
+  set "PLATFORM=x64"
+)
+
+
+
+:: These are here to map cl.exe version numbers, which we use to figure out which
+::   compiler we are using, and which compiler consumers of Qt need to use, to MSVC
```
Comment (ccordoba12) [inline svn/bld.bat]: Typo here:

> ... which compiler consumers of **Qt** need to use ...

File: svn/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  url: http://www.trieuvan.com/apache/subversion/subversion-1.9.3.tar.bz2
+  sha1: 27e8df191c92095f48314a415194ec37c682cbcf
+  patches:
+    # - swig_regex.patch    [win and py>30]
+    #- swig_unicode_2.patch    [win and py>30]
+    #- swig_file_mode.patch    [win and py>30]
+    #- swig_init.patch    [win and py>30]
+    #- swig_gen_base.patch    [win and py>30]
+    #- swig_gen_win.patch    [win and py>30]
+    #- swig_external.patch    [win and py>30]
```
Comment (ccordoba12) [inline svn/meta.yaml]: Why not remove these comments?

Are these patchs present in the tree too? If that's the case, why not remove them too?

File: swig/meta.yaml [hunk 3]
```diff
@@ -3,19 +3,22 @@ package:
...
-  md5: 7fff46c84b8c630ede5b0f0827e3d90a
+  fn: swig-3.0.7.tar.gz   [unix]
+  url: http://prdownloads.sourceforge.net/swig/swig-3.0.7.tar.gz   [unix]
+  md5: 7fff46c84b8c630ede5b0f0827e3d90a  [unix]
+  fn: swigwin-3.0.7.zip   [win]
+  url: http://prdownloads.sourceforge.net/swig/swigwin-3.0.7.zip   [win]
+  sha256: 11423c4a37209ea501d04d2bf7f71c275cf418b93a28cf062c87b3c1fdfbe286 [win]
 
 build:
   detect_binary_files_with_prefix: True
```
Comment (ccordoba12) [inline swig/meta.yaml]: Is this necessary for Windows too? It might be not :-)

**Comment by ccordoba12** [comment]
Quick review done!

**Comment by msarahan** [comment]
Thanks, @ccordoba12 - fixed your suggestions; merging.

**Pull request merged**